extensions/datasketches/src/main/java/io/druid/query/aggregation/datasketches/theta/SketchMergeComplexMetricSerde.java

```diff
@@ -67,11 +65,10 @@ public Object extractValue(InputRow inputRow, String metricName)
   }
 
   @Override
-  public ColumnPartSerde deserializeColumn(ByteBuffer buffer, ColumnBuilder builder)
+  public void deserializeColumn(ByteBuffer buffer, ColumnBuilder builder)
```

Comment:
since we are changing the behavior of this method, can we please add a comment on the interface about how the method is supposed to be used?